here-naksha-lib-view/src/main/java/com/here/naksha/lib/view/ViewReadSession.java

```diff
@@ -206,74 +197,72 @@ private boolean isRequestOnlyById(ReadRequest request) {
 
   @Override
   public int getSocketTimeout() {
-    //TODO
+    // TODO
```

Comment:
I know this is not the scope of this PR - but there's a ton of TODOs.
Where (which ticket) will it be done?